.github/workflows/build-docker-internal.yml

```diff
@@ -209,7 +209,7 @@ jobs:
             curl -u${{ secrets.JFROG_USERNAME }}:${{ secrets.JFROG_PASSWORD }} -XPUT "https://${{ secrets.JFROG_REGISTRY }}/hpccpl-debian-local/pool/LN/${_package};deb.distribution=${distribution};deb.component=LN;deb.architecture=amd64" -T ${{ steps.vars.outputs.folder_build }}/${_package} 
           done
   
-      - name: Upload Assets to Jfrog (centos)
+      - name: Upload Assets to Jfrog
```

Comment:
why? So this step in the job is specific to uploading .rpm packages to the hpccpl-rpm-local repository. The step above it is specific to the .deb packages.  The two repository types have a different folder layout which is why they're broken up into two different steps.

Reply:
That change was not meant to be in there.  Tweaked to avoid confusion with CentOS and Rocky...